dqc_us_rules/dqc_us_0001.py

```diff
@@ -123,7 +133,8 @@ def _run_member_checks(axis, axis_config, relset, val, role):
                     child.qname.localName,
                     val.modelXbrl
                 )
-                if len(fact_list) != 0:
+                if (len(fact_list) != 0 and
+                    (axis.qname.localName, child.qname.localName) not in checked_axes[axis_key]):
```

Comment:
1. I'm having thoughts that maybe qname would be better (in the extremely unlikely event that an extension had the same local name as a gaap concept 
2. The tuple is long and repeated 6 times, if there's a variable name that makes sense, I think it'd look better/shorter to extract it to a var right after fact_list is built in both check functions so it was only defined once per check function, 2 times total